tests/foreman/cli/test_errata.py

```diff
@@ -1275,3 +1280,112 @@ def test_positive_check_errata_dates(module_org):
     # Verify any errata UPDATED date from stdout
     validate_updated_date = datetime.strptime(result[0]['updated'], '%Y-%m-%d').date()
     assert isinstance(validate_updated_date, date)
+
+
+@pytest.fixture(scope='module')
+def rh_repo_module_manifest(module_manifest_org):
+    """Use module manifest org, creates RH tools repo, syncs and returns RH repo."""
+    # enable rhel repo and return its ID
+    rh_repo_id = enable_rhrepo_and_fetchid(
+        basearch=DEFAULT_ARCHITECTURE,
+        org_id=module_manifest_org.id,
+        product=PRDS['rhel'],
+        repo=REPOS['rhst7']['name'],
+        reposet=REPOSET['rhst7'],
+        releasever=None,
+    )
+    # Sync step because repo is not synced by default
+    rh_repo = entities.Repository(id=rh_repo_id).read()
+    rh_repo.sync()
+    return rh_repo
+
+
+"""Section for tests using RHEL7.7 Content Host.
+   The applicability tests using Default Content View are related to the introduction of Pulp3.
+   """
+
+
+@pytest.fixture(scope='module')
+def default_contentview(module_manifest_org):
+    return entities.ContentView(organization=module_manifest_org, name=DEFAULT_CV).search()
+
+
+@pytest.fixture(scope='module')
+def new_module_ak(module_manifest_org, rh_repo_module_manifest, default_lce):
+    new_module_ak = entities.ActivationKey(
+        content_view=module_manifest_org.default_content_view,
+        environment=entities.LifecycleEnvironment(id=module_manifest_org.library.id),
+        organization=module_manifest_org,
+    ).create()
+    # Ensure tools repo is enabled in the activation key
+    new_module_ak.content_override(
+        data={'content_overrides': [{'content_label': REPOS['rhst7']['id'], 'value': '1'}]}
+    )
+    # Fetch available subscriptions
+    subs = entities.Subscription(organization=module_manifest_org).search()
+    assert len(subs) > 0
+    # Add default subscription to activation key
+    sub_found = False
+    for sub in subs:
+        if sub.name == DEFAULT_SUBSCRIPTION_NAME:
+            new_module_ak.add_subscriptions(data={'subscription_id': sub.id})
+            sub_found = True
+    assert sub_found
+    return new_module_ak
+
+
+@pytest.fixture
+def chost(module_manifest_org, rhel77_contenthost_module, new_module_ak):
+    """A RHEL77 Content Host that has applicable errata and registered to Library"""
+    # python-psutil is obsoleted by python2-psutil, so install older python2-psutil for errata test
+    rhel77_contenthost_module.run(
+        'rpm -Uvh https://download-ib01.fedoraproject.org/pub/epel/7/'
+        'x86_64/Packages/p/python2-psutil-5.6.7-1.el7.x86_64.rpm'
+    )
+    rhel77_contenthost_module.install_katello_ca()
+    rhel77_contenthost_module.register_contenthost(module_manifest_org.label, new_module_ak.name)
+    host_id = rhel77_contenthost_module.nailgun_host.id
+    host_content = entities.Host(id=host_id).read_json()
+    assert host_content['subscription_status'] == 0
```

Comment:
Is this not already an attribute of the nailgun_host object?

Reply:
@JacobCallahan I don't see it:
>>> result = rhel77_contenthost_module.nailgun_host
>>> result
nailgun.entities.Host(architecture=nailgun.entities.Architecture(id=1), build=False, capabilities=['build'], comment=None, compute_profile=None,     
compute_resource=None, content_facet_attributes={'id': 2, 'uuid': '165d5c6d-0bbb-49c1-9609-8c70161ef80e', 'content_view_id': 4, 'content_view_name': 
'Default Organization View', 'lifecycle_environment_id': 4, 'lifecycle_environment_name': 'Library', 'content_source_id': None,                      
'content_source_name': None, 'kickstart_repository_id': None, 'kickstart_repository_name': None, 'errata_counts': {'security': 0, 'bugfix': 0,       
'enhancement': 0, 'total': 0}, 'applicable_package_count': 0, 'upgradable_package_count': 0, 'applicable_module_stream_count': 0,                    
'upgradable_module_stream_count': 0, 'content_view': {'id': 4, 'name': 'Default Organization View'}, 'lifecycle_environment': {'id': 4, 'name':      
'Library'}, 'content_source': None, 'kickstart_repository': None}, domain=nailgun.entities.Domain(id=1), enabled=True, environment=None,             
hostgroup=None, image=None, ip='10.1.3.225', location=nailgun.entities.Location(id=2), mac='56:6f:0d:72:00:2e', managed=False, medium=None,          
model=nailgun.entities.Model(id=1), name='dhcp-3-225.vms.sat.rdu2.redhat.com', operatingsystem=nailgun.entities.OperatingSystem(id=2),               
organization=nailgun.entities.Organization(id=5), owner=nailgun.entities.User(id=1), provision_method='build', ptable=None, puppet_ca_proxy=None,    
puppet_proxy=None, realm=None, subnet=None, uuid=None, id=3, owner_type='User')

Reply:
Agreed, creation of `host_content` from `entities.Host` is redundant here. `rhel77_contenthost_module.nailgun_host` is the same entity as `entities.Host(id=host_id)`.

You should at the least be able to just directly call `read_json()` on `rhel77_contenthost_module`

Reply:
OK, this works:
`In [8]: rhel77_contenthost_module.nailgun_host.read_json()['subscription_status']`
`Out[8]: 0`